docs/hkdb.rst

```diff
@@ -0,0 +1,102 @@
+
+==============================
+Loading L3 Houskeeping Data
+==============================
+
+The ``sotodlib.io.hkdb`` module provides methods for indexing, and quickly
+loading housekeeping data from an archive.
+The databases implemented store the following information:
+
+- List of all housekeeping files, with start and stop times
+- List of every frame in the file, with the start and stop times of each frame,
+  agent instance-id, and the byte-offset of the frame in the file.
+
+To index and load housekeeping information, you will need to create a
+``HkConfig`` object, documented in the API section below. For example, the
+following yaml file can be loaded in directly with ``HkConfig.from_yaml``:
+
+.. code-block:: yaml
+
+    hk_root: /so/data/satp1/hk
+    hk_db: satp1_hk.db
+    aliases:
+        fp_temp: cryo-ls372-lsa21yc.temperatures.Channel_02_T
+
+In addition to the hk-root and database paths, an "alias" dictionary can
```

Comment:
Mention that aliases are pertinent to _load_ only, and don't affect how info is stored in the database.

Might be helpful to have an example where a common database configuration is loaded from file, but user modifies (in script) the aliases block to suit their current purpose.  (I.e. emphasize that aliases are a dynamic element to tune for each use case, whereas a database instance is a general storage tool useful for many purposes.)